zeppelin/Dockerfile

```diff
@@ -1,12 +1,167 @@
 # Image: pubnative/zeppelin
 
-FROM dylanmei/zeppelin:0.7.3
+FROM debian:stretch
 
-RUN echo "deb http://repos.mesosphere.io/debian/ jessie main" > /etc/apt/sources.list.d/mesosphere.list \
-    && apt-key adv --keyserver keyserver.ubuntu.com --recv E56151BF \
-    && apt-get -y update \
-    && apt-get -y install mesos \
-    && apt-get clean \
-    && rm -rf /var/lib/apt/lists/*
+RUN apt-get update \
+ && apt-get install -y locales \
+ && dpkg-reconfigure -f noninteractive locales \
+ && locale-gen C.UTF-8 \
+ && /usr/sbin/update-locale LANG=C.UTF-8 \
+ && echo "en_US.UTF-8 UTF-8" >> /etc/locale.gen \
+ && locale-gen \
+ && apt-get clean \
+ && rm -rf /var/lib/apt/lists/*
 
-RUN ./bin/install-interpreter.sh --name file,livy
+# Users with other locales should set this in their derivative image
+ENV LANG en_US.UTF-8
+ENV LANGUAGE en_US:en
+ENV LC_ALL en_US.UTF-8
+
+RUN apt-get update \
+ && apt-get install -y curl unzip \
```

Comment:
I suppose it should be 2 spaces indentation

Reply:
👍